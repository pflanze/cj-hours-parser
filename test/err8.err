Error: can't process file "test/err8.txt"

Caused by:
    line 4, column 18: missing '-->' after '<!--'
