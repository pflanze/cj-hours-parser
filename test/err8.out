item=Other([Time(NT(03:27:00)), Minus, Time(NTZ(10:44:10, Zone("GMT"))), Rest("<-- --> test-again")])
    =Other([Subitem(L(0, 1, 0), Time(NT(03:27:00))), Subitem(L(0, 1, 5), Minus), Subitem(L(0, 1, 6), Time(NTZ(10:44:10, Zone("GMT")))), Subitem(L(0, 1, 18), Rest("<-- --> test-again"))])
item=Other([Time(NT(03:27:00)), Minus, Time(NTZ(10:44:10, Zone("GMT"))), Rest(" test-again")])
    =Other([Subitem(L(0, 2, 0), Time(NT(03:27:00))), Subitem(L(0, 2, 5), Minus), Subitem(L(0, 2, 6), Time(NTZ(10:44:10, Zone("GMT")))), Subitem(L(0, 2, 26), Rest(" test-again"))])
item=Other([Time(NT(03:27:00)), Minus, Time(NTZ(10:44:10, Zone("GMT"))), Rest(" test-again")])
    =Other([Subitem(L(0, 3, 0), Time(NT(03:27:00))), Subitem(L(0, 3, 5), Minus), Subitem(L(0, 3, 6), Time(NTZ(10:44:10, Zone("GMT")))), Subitem(L(0, 3, 25), Rest(" test-again"))])
